src/SFA.DAS.Forecasting.Web/Extensions/HtmlHelperExtensions.cs

```diff
@@ -20,6 +21,44 @@ public static class HtmlHelperExtensions
             }
 
             return new MvcHtmlString("error");
-        }   
+        }
+
+
+        public static MvcHtmlString SetZenDeskLabels(this HtmlHelper html, params string[] labels)
+        {
+            var apiCallString =
+                "<script type=\"text/javascript\">zE('webWidget', 'helpCenter:setSuggestions', { labels: [";
+
+            var first = true;
+            foreach (var label in labels)
+            {
+                if (!string.IsNullOrEmpty(label))
+                {
+                    if (!first) apiCallString += ",";
+                    first = false;
```

Comment:
Personal preference but I find it easier to read If statements not on a single line - could also include the setting of the `isFirstLabel` to be inside there